examples/platform/silabs/SiWx917/BUILD.gn

```diff
@@ -52,9 +52,8 @@ declare_args() {
 assert(chip_enable_wifi)
 
 silabs_common_plat_dir = "${chip_root}/examples/platform/silabs"
-silabs_plat_si91x_wifi_dir = "${chip_root}/src/platform/silabs/SiWx917/wifi"
+silabs_plat_si91x_wifi_dir = "${chip_root}/src/platform/silabs"
```

Comment:
`silabs_plat_si91x_wifi_dir` does not seem to point to the directory which the variable indicates.

`silabs_plat_dir` might be more suited for this location.

Reply:
Changed the name of the variable.